docs/tutorials/depth/videos_monodepth2.py

```diff
@@ -0,0 +1,196 @@
+scale_factor = 5.4
+MIN_DEPTH = 1e-3
+MAX_DEPTH = 80
+
+feed_height = 192
+feed_width = 640
+
+pred_depth_sequences = []
+pred_disp_sequences = []
+for img in raw_img_sequences:
+    img = img.resize((feed_width, feed_height), pil.LANCZOS)
+    img = transforms.ToTensor()(mx.nd.array(img)).expand_dims(0).as_in_context(context=ctx)
+
+    outputs = model.predict(img)
+    mx.nd.waitall()
+    pred_disp, _ = disp_to_depth(outputs[("disp", 0)], min_depth, max_depth)
+    t = time.time()
+    pred_disp = pred_disp.squeeze().as_in_context(mx.cpu()).asnumpy()
+    pred_disp = cv2.resize(src=pred_disp, dsize=(original_width, original_height))
+    pred_disp_sequences.append(pred_disp)
+
+    pred_depth = 1 / pred_disp
+    pred_depth *= scale_factor
+    pred_depth[pred_depth < MIN_DEPTH] = MIN_DEPTH
+    pred_depth[pred_depth > MAX_DEPTH] = MAX_DEPTH
+    pred_depth_sequences.append(pred_depth)
+
+
+##############################################################################
+# Store results
+# -----------------
+#
+# Here, we provide an example of storing the prediction results. Including:
+#
+# - store depth map
+output_path = os.path.join(os.path.expanduser("."), "tmp")
+
+pred_path = os.path.join(output_path, 'pred')
+if not os.path.exists(pred_path):
+    os.makedirs(pred_path)
+
+for pred, file in zip(pred_depth_sequences, files):
+    pred_out_file = os.path.join(pred_path, file)
+    cv2.imwrite(pred_out_file, pred)
+
+##############################################################################
+# - store disparity and save it to a video
+rgb_path = os.path.join(output_path, 'rgb')
+if not os.path.exists(rgb_path):
+    os.makedirs(rgb_path)
+
+output_sequences = []
+for raw_img, pred, file in zip(raw_img_sequences, pred_disp_sequences, files):
+    vmax = np.percentile(pred, 95)
+    normalizer = mpl.colors.Normalize(vmin=pred.min(), vmax=vmax)
+    mapper = cm.ScalarMappable(norm=normalizer, cmap='magma')
+    colormapped_im = (mapper.to_rgba(pred)[:, :, :3] * 255).astype(np.uint8)
+    im = pil.fromarray(colormapped_im)
+
+    raw_img = np.array(raw_img)
+    pred = np.array(im)
+    output = np.concatenate((raw_img, pred), axis=0)
+    output_sequences.append(output)
+
+    pred_out_file = os.path.join(rgb_path, file)
+    cv2.imwrite(pred_out_file, cv2.cvtColor(pred, cv2.COLOR_RGB2BGR))
+
+width = int(output_sequences[0].shape[1] + 0.5)
+height = int(output_sequences[0].shape[0] + 0.5)
+fourcc = cv2.VideoWriter_fourcc(*'mp4v')
+out = cv2.VideoWriter(
+    os.path.join(output_path, 'demo.mp4'), fourcc, 20.0, (width, height))
+
+for frame in output_sequences:
+    frame = cv2.cvtColor(frame, cv2.COLOR_RGB2BGR)
+
+    out.write(frame)
+    cv2.imshow('demo', frame)
+
+    if cv2.waitKey(25) & 0xFF == ord('q'):
+        break
+
+##############################################################################
+# We release the webcam before exiting:
+out.release()
+cv2.destroyAllWindows()
+
```

Comment:
Please provide the generated video demo here, so that users can see what are the results. You can transform the .mp4 file into a .gif and embed it here.

Reply:
I have provided a gif image, thanks for your suggestions！